pkgs/development/tools/rust/cargo-pgx/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib, stdenv, fetchFromGitHub, rustPlatform, pkg-config, openssl, libiconv, Security }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "cargo-pgx";
+  version = "0.4.5";
+
+  src = fetchFromGitHub {
+    owner = "tcdi";
+    repo = "pgx";
+    rev = "v${version}";
+    sha256 = "sha256-M81n4ORhMYpXRonJxEgFMOuR0Deb5CN2df70ltviIqI=";
+  };
```

Comment:
Might be best to use `fetchCrate` here, since `cargo-pgx` is published as a separate crate.

Reply:
done